<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="2836" started-at="2019-12-16T23:35:08Z" finished-at="2019-12-16T23:35:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="2836" started-at="2019-12-16T23:35:08Z" finished-at="2019-12-16T23:35:11Z">
      <class name="sachin.TestSample">
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String)[pri:0, instance:sachin.TestSample@5034c75a]" name="test" duration-ms="2771" started-at="2019-12-16T23:35:08Z" data-provider="getdata" finished-at="2019-12-16T23:35:11Z">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
Data provider mismatch
Method: test([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.util.ArrayList) [Adithya, Geejagaru, sdkjwd, x;laksjl, 2136598753, asldjalk],(java.util.ArrayList) [Adithya, Geejagaru, sdkjwd, x;laksjl, 2136598753, asldjalk],null,null,null,null,null,null,null,null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: test([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.util.ArrayList) [Adithya, Geejagaru, sdkjwd, x;laksjl, 2136598753, asldjalk],(java.util.ArrayList) [Adithya, Geejagaru, sdkjwd, x;laksjl, 2136598753, asldjalk],null,null,null,null,null,null,null,null]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:45)
at org.testng.internal.Parameters.injectParameters(Parameters.java:796)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:983)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- sachin.TestSample -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
